packages/composite-checkout-wpcom/src/components/wp-checkout-order-review.js

```diff
@@ -18,7 +18,7 @@ import {
 	WPOrderReviewSection,
 } from './wp-order-review-line-items';
 
-export default function WPCheckoutOrderReview( { className } ) {
+export function WPCheckoutOrderReview( { className, removeItem } ) {
```

Comment:
I don't mind, but is there a reason to change this to a named export rather than the default? I don't see other exports...